libs/sitemaps/tsconfig.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
This tsconfig should only extend `/libs/tsconfig.json` and then we should create another `tsconfig.build.json` file which defines the build options. The tsconfigs should be identical to the tsconfigs in `/libs/models`.

Reply:
Yup... resolved!

Reply:
This tsconfig should be empty leaving only the `extends` option.

Reply:
With your current configuration, `yarn check-types` fails because your tsconfig has `rootDir: src` and `/test` is outside of that.